src/app/shared/menu/menu.service.ts

```diff
@@ -79,13 +80,22 @@ const getSubSectionsFromSectionSelector = (id: string): MemoizedSelector<MenuSta
   return menuKeySelector<string[]>(id, menuSectionIndexStateSelector);
 };
 
+export const PINNED_MENU_COOKIE = 'dsMenuCollapsedState';
```

Comment:
@gingyx : Just quick feedback on this.  Any new cookies need to be added to the "Cookie Settings" (in UI footer) via Orejime.  We also need to offer users the ability to disable any cookies which are not required. (Obviously if they disable the cookie though, then this feature will not be available.)